packages/firestore/src/local/lru_garbage_collector.ts

```diff
@@ -265,13 +267,19 @@ export class LruScheduler implements GarbageCollectionScheduler {
     this.gcTask = this.asyncQueue.enqueueAfterDelay(
       TimerId.LruGarbageCollection,
       delay,
-      () => {
+      async () => {
         this.gcTask = null;
         this.hasRun = true;
-        return localStore
-          .collectGarbage(this.garbageCollector)
-          .then(() => this.scheduleGC(localStore))
-          .catch(ignoreIfPrimaryLeaseLoss);
+        try {
+          await localStore.collectGarbage(this.garbageCollector);
+        } catch (e) {
+          if (e.name === 'IndexedDbTransactionError') {
+            logDebug(LOG_TAG, 'Ignoring IndexedDB error during LRU: ', e);
```

Comment:
"LRU" is not a great descriptor of what's going on here. How about "garbage collection"?

Also update the PR description.

Reply:
Done